app.py

```
@@ -2088,10 +2090,25 @@ async def api_weekly_menu(req: WHOTargetsRequest) -> WeeklyMenuResponse:
         }:
             raise HTTPException(status_code=503, detail="VIP module is disabled")
 
-        # Use globals() instead of sys.modules to access module-level make_weekly_menu
-        # This works correctly when app.py is loaded dynamically by app/__init__.py
-        _make_weekly_menu = globals().get("make_weekly_menu")
-        if _make_weekly_menu is None:
+        import sys as _sys
+
+        module_name = "core.menu_engine"
+        modules_registry = getattr(_sys, "modules")
```

Ruff (0.14.0)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)